packages/actor-scraper/jsdom-scraper/src/internals/crawler_setup.ts

```diff
@@ -158,6 +158,8 @@ export class CrawlerSetup implements CrawlerSetupOptions {
             proxyConfiguration: this.proxyConfiguration,
             requestHandler: this._requestHandler.bind(this),
             preNavigationHooks: [],
+            runScripts: this.input.runScripts ?? false,
+            hideInternalConsole: !this.input.showInternalConsole ?? true,
```

Comment:
the defaults in the input schema are saying the opposite, so i guess it should be like this:
```suggestion
            runScripts: this.input.runScripts ?? true,
            hideInternalConsole: !this.input.showInternalConsole ?? false,
```

Reply:
Huh, I swear the `!` vs `??` precedence seemed better in my head :)